archive-product.php

```diff
@@ -7,43 +7,51 @@
 <main id="main-content">
 
   <!-- main posts loop -->
-  <section id="posts" class="row">
-
+  <section id="products" class="container">
+    <div class="row margin-bottom-small">
 <?php
 if( have_posts() ) {
+  $i = 0;
   while( have_posts() ) {
     the_post();
 
     $product = new WC_Product($post->ID);
     $in_stock = $product->is_in_stock();
     $availability = $product->get_availability();
-?>
-
-    <article <?php post_class('col col3'); ?> id="post-<?php the_ID(); ?>">
-
-      <?php if (!$in_stock) { ?>
-      <span class="sold"><?php echo $availability['availability']; ?></span>
-      <?php } ?>
-
-      <?php the_post_thumbnail(); ?>
-
-      <a href="<?php the_permalink() ?>"><?php the_title(); ?></a>
 
+    if ($i % 4 === 0 && $i !== 0) {
+      echo "</div>\n<div class=\"row margin-bottom-small\">";
+    }
+?>
+    <article <?php post_class('col col3 shop-product'); ?> id="product-<?php the_ID(); ?>">
+      <a href="<?php the_permalink() ?>">
+        <div class="shop-product-title-holder">
```

Comment:
u and i should decide on class naming convention about `-container` v. `-holder`.  i hardly ever put holder but ive noticed u use it and i like it. can we come up with a rule for when to use container and when to use holder?